main.tf

```diff
@@ -84,6 +84,19 @@ module "database" {
   depends_on          = [module.project_factory_project_services]
 }
 
+module "redis" {
+  count          = var.create_redis ? 1 : 0
+  source         = "./modules/redis"
+  namespace      = var.namespace
+  memory_size_gb = 4
+  network        = module.networking.network
+
+}
+
+locals {
+  redis_connection_string = var.create_redis ? "redis://${module.redis.0.connection_string}?tls=true&ttlInSeconds=60" : null
```

Comment:
nice, looks like the maxmemory policy for memorystore is `volatile-lru` so we'll need the TTL here too